### PR TITLE
HOSS2-170 - Make messages file standardised

### PR DESCRIPTION
**Pull request by burns238**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
+lookup.givenName.label='First name'
+lookup.givenName.ariaDescribedByError='givenName-error'
+lookup.familyName.label='Surname'
+lookup.familyName.ariaDescribedByError='familyName-error'
+lookup.dateOfBirth.label='Date of birth'
+lookup.dateOfBirth.hint='For example, 31 3 1980'
+lookup.inputMode='numeric'
+lookup.hintDOBid='dob-hint'
+lookup.hintDOBIderror='dateOfBirth-error'
+date.day =' Day'
```
Comment (liampan) [inline conf/messages:59]: this isnt correct

**Review by liampan — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 1 above)
Comment (AdamCoops) [inline conf/messages:59]: Are the Day/Month/Year supposed to have a space at the front? I know they used to, it's just not aligned with the rest of the file.

File: conf/messages [hunk 2]
```diff
@@ -1,64 +1,64 @@
...
+error.year.invalid-format='Enter a year of birth'
+error.day.invalid-format='Enter a day of birth'
+error.month.invalid-format='Enter a month of birth'
+error.nino.required='Enter National Insurance number in the correct format'
+error.nino.invalid-format='Invalid National Insurance number'
+error.familyName.required='Enter surname'
+error.familyName.invalid-format='Invalid surname'
+error.givenName.required='Enter first name'
+error.givenName.invalid-format='Invalid first name'
+error.browser.title.prefix =' Error:'
```
Comment (liampan) [inline conf/messages:34]: same here

File: conf/messages [hunk 3]
```diff
@@ -1,64 +1,64 @@
 # ALPHA Banner
 # ----------------------------------------------------------
-feedback.before = This is a new service. Your
-feedback.link = feedback
-feedback.after = will help us to improve it.
+feedback.before =' This is a new service. Your'
```
Comment (AdamCoops) [inline conf/messages:3]: whitespace?

**Review by liampan — APPROVED** [review]

**Review by liampan — APPROVED** [review]

**Pull request merged**